app/src/pages/inside/membersPage/membersPage.jsx

```diff
@@ -188,18 +184,24 @@ export class MembersPage extends Component {
             onInvite={this.inviteUser}
           />
           <MembersGrid data={members} fetchData={this.props.fetchMembersAction} loading={loading} />
-          {this.props.filter !== null && this.props.filter !== ''
-            ? !loading && this.getNoFiltersFound()
-            : !loading && (
-                <PaginationToolbar
-                  activePage={activePage}
-                  itemCount={itemCount}
-                  pageCount={pageCount}
-                  pageSize={pageSize}
-                  onChangePage={onChangePage}
-                  onChangePageSize={onChangePageSize}
+          {!loading &&
+            (members.length ? (
+              <PaginationToolbar
+                activePage={activePage}
+                itemCount={itemCount}
+                pageCount={pageCount}
+                pageSize={pageSize}
+                onChangePage={onChangePage}
+                onChangePageSize={onChangePageSize}
+              />
+            ) : (
+              this.props.filter && (
```

Comment:
Too many nested conditions. Please simplify it. Maybe it can be split into smaller functions

Reply:
Fixed